.github/workflows/ci.yml

```
@@ -43,45 +43,19 @@ jobs:
+      id: changesets
+      uses: changesets/action@v1
       with:
-        token: ${{ secrets.NPM_TOKEN }}
-        access: public
\ No newline at end of file
+        # This expects you to have a script called `release` in your package.json
+        # that runs `yarn build && changeset publish`. We added this earlier.
+        publish: yarn release
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)